modules/network-observability-operator-install.adoc

```diff
@@ -9,7 +9,7 @@ You can install the Network Observability Operator using the {product-title} web
 
 .Prerequisites
 
-* Installed Loki. It is recommended to install Loki using the link:https://catalog.redhat.com/software/containers/openshift-logging/loki-rhel8-operator/622b46bcae289285d6fcda39[Loki Operator version 5.7].
+* It is recommended to install Loki using the link:https://catalog.redhat.com/software/containers/openshift-logging/loki-rhel8-operator/622b46bcae289285d6fcda39[Loki Operator version 5.7].
 * One of the following supported architectures is required: `amd64`, `ppc64le`, `arm64`, or `s390x`.
 * Any CPU supported by Red Hat Enterprise Linux (RHEL) 9
 
```

Comment:
Need to add before L40 in this doc below to also set loki.enable to true when using loki

Reply:
With the [easier config](https://issues.redhat.com/browse/NETOBSERV-764), is this still true? In https://github.com/openshift/openshift-docs/pull/63276, I'm removing that piece for 1.4. cc @jpinsonneau

Reply:
Yes, the `enable` flag will still be there even after easier config work to indicate if you are using loki or not.

So as Amogh suggested, it would be good to add above `loki.url` explanation a `loki.enable` note.

Reply:
Oh okay, I think I was actually confusing this with loki.tls.enable. Crucially, I realize that this ask from Amogh is just loki.enable, I'm sorry!!

Reply:
Should the `loki.enable` just say that it should be set to true? Additionally, I was thinking that the `loki.url` and `status.url` should be removed. I guess I'm getting this from this [doc](https://docs.google.com/document/d/1yx0_wQBs84GbDOWmzVecbkeWLO4OymtWkVJqOX6JURk/edit#heading=h.xvi37l7v2hsd). I think I need a total clarification about what the updated LokiStack CRD should look like
 `loki:                                       
    url: 'https://loki-gateway-http.netobserv.svc:8080/api/logs/v1/network'
    statusUrl: 'https://loki-query-frontend-http.netobserv.svc:3100/'
    authToken: FORWARD`

Reply:
Update looks good, thanks!